pkg/client/containerinfo_test.go

```diff
@@ -28,8 +28,8 @@ import (
 	"testing"
 	"time"
 
-	"github.com/google/cadvisor/info"
-	itest "github.com/google/cadvisor/info/test"
+	info "github.com/google/cadvisor/info/v1"
+	itest "github.com/google/cadvisor/info/v1/test"
```

Comment:
Not a fan of the name either, but this PR only updates the dep. We can change it in another PR if we'd like :)